project/AkkaDisciplinePlugin.scala

```diff
@@ -132,6 +108,8 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
     "-Ywarn-unused:_",
     "-Ypartial-unification",
     "-Ywarn-extra-implicit",
-  )
+    "akka-protobuf",
```

Comment:
Huh? merge mistake?

Reply:
Yeah, there was something that messed up merging this in all my PRs